scss/forms/_floating-labels.scss

```diff
@@ -0,0 +1,59 @@
+// stylelint-disable selector-no-vendor-prefix
+
+.form-floating {
+  position: relative;
+
+  > label,
+  > .form-control {
+    height: $form-floating-height;
+    padding: $form-floating-padding-y $form-floating-padding-x;
+  }
+
+  > label {
+    position: absolute;
+    top: 0;
+    left: 0;
+    display: block;
+    width: 100%;
+    margin-bottom: 0; // Override default `<label>` margin
+    // color: inherit;
+    pointer-events: none;
+    cursor: text; // Match the input under the label
+    border: $input-border-width solid transparent;
+    @include border-radius(.25rem);
+    @include transition(all .1s ease-in-out);
+  }
+
+  > .form-control {
+    &::-webkit-input-placeholder  { color: transparent; }
+    &::-ms-input-placeholder      { color: transparent; }
+    &::-moz-placeholder           { color: transparent; }
+    &::placeholder                { color: transparent; }
```

Comment:
I think autoprefixer already prefixes these?

Reply:
Yup and makes me wonder why Stylelint doesn't error out...